### ISSUE TITLE
The `idle_time_before_shutdown_minutes` setting mentioned in the docs is not respected when trying to create a new compute using the latest version of az ml

### ISSUE THREAD

**Issue by vladiliescu** [Azure/azure-cli-extensions#5585]
## The `idle_time_before_shutdown_minutes` setting mentioned in the docs is not respected when trying to create a new compute using the latest version of az ml

**Command Name**
`az ml compute create
Extension Name: ml. Version: 2.11.0.`

**Errors:**
```
The command failed with an unexpected error. Here is the traceback:
Validation for ComputeInstanceSchema failed:

 {
  "idle_time_before_shutdown_minutes": [
    "Unknown field."
  ]
} 

 
Visit this link to refer to the ComputeInstance schema if needed: https://aka.ms/ml-cli-v2-compute-aml-yaml-reference.
The easiest way to author a specification file is using IntelliSense and auto-completion Azure ML VS code extension provides: https://code.visualstudio.com/docs/datascience/azure-machine-learning. To set up: https://docs.microsoft.com/azure/machine-learning/how-to-setup-vs-code
Traceback (most recent call last):
  File "/Users/vladiliescu/.azure/cliextensions/ml/azext_mlv2/manual/vendored_curated_sdk/azure/ai/ml/entities/_util.py", line 148, in load_from_dict
    return schema(context=context).load(data, **kwargs)
  File "/Users/vladiliescu/.azure/cliextensions/ml/marshmallow/schema.py", line 722, in load
    return self._do_load(
  File "/Users/vladiliescu/.azure/cliextensions/ml/marshmallow/schema.py", line 909, in _do_load
    raise exc
marshmallow.exceptions.ValidationError: {'idle_time_before_shutdown_minutes': ['Unknown field.']}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/Cellar/azure-cli/2.42.0/libexec/lib/python3.10/site-packages/knack/cli.py", line 233, in invoke
    cmd_result = self.invocation.execute(args)
  File "/usr/local/Cellar/azure-cli/2.42.0/libexec/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 663, in execute
    raise ex
  File "/usr/local/Cellar/azure-cli/2.42.0/libexec/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 726, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
  File "/usr/local/Cellar/azure-cli/2.42.0/libexec/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 697, in _run_job
    result = cmd_copy(params)
  File "/usr/local/Cellar/azure-cli/2.42.0/libexec/lib/python3.10/site-packages/azure/cli/core/commands/__init__.py", line 333, in __call__
    return self.handler(*args, **kwargs)
  File "/usr/local/Cellar/azure-cli/2.42.0/libexec/lib/python3.10/site-packages/azure/cli/core/commands/command_operation.py", line 121, in handler
    return op(**command_args)
  File "/Users/vladiliescu/.azure/cliextensions/ml/azext_mlv2/manual/custom/compute.py", line 140, in ml_compute_create
    log_and_raise_error(e, debug)
  File "/Users/vladiliescu/.azure/cliextensions/ml/azext_mlv2/manual/custom/raise_error.py", line 145, in log_and_raise_error
    raise cli_error
  File "/Users/vladiliescu/.azure/cliextensions/ml/azext_mlv2/manual/custom/compute.py", line 130, in ml_compute_create
    compute = load_compute(source=file, params_override=params_override)
  File "/Users/vladiliescu/.azure/cliextensions/ml/azext_mlv2/manual/vendored_curated_sdk/azure/ai/ml/entities/_load_functions.py", line 318, in load_compute
    return load_common(Compute, source, relative_origin, **kwargs)
  File "/Users/vladiliescu/.azure/cliextensions/ml/azext_mlv2/manual/vendored_curated_sdk/azure/ai/ml/entities/_load_functions.py", line 105, in load_common
    raise e
  File "/Users/vladiliescu/.azure/cliextensions/ml/azext_mlv2/manual/vendored_curated_sdk/azure/ai/ml/entities/_load_functions.py", line 89, in load_common
    return _load_common_raising_marshmallow_error(cls, yaml_dict, relative_origin, params_override, **kwargs)
  File "/Users/vladiliescu/.azure/cliextensions/ml/azext_mlv2/manual/vendored_curated_sdk/azure/ai/ml/entities/_load_functions.py", line 135, in _load_common_raising_marshmallow_error
    return cls._load(data=yaml_dict, yaml_path=relative_origin, params_override=params_override, **kwargs)
  File "/Users/vladiliescu/.azure/cliextensions/ml/azext_mlv2/manual/vendored_curated_sdk/azure/ai/ml/entities/_compute/compute.py", line 180, in _load
    return ComputeInstance._load_from_dict(data, context, **kwargs)
  File "/Users/vladiliescu/.azure/cliextensions/ml/azext_mlv2/manual/vendored_curated_sdk/azure/ai/ml/entities/_compute/compute_instance.py", line 336, in _load_from_dict
    loaded_data = load_from_dict(ComputeInstanceSchema, data, context, **kwargs)
  File "/Users/vladiliescu/.azure/cliextensions/ml/azext_mlv2/manual/vendored_curated_sdk/azure/ai/ml/entities/_util.py", line 151, in load_from_dict
    raise ValidationError(decorate_validation_error(schema, pretty_error, additional_message))
marshmallow.exceptions.ValidationError: Validation for ComputeInstanceSchema failed:

 {
  "idle_time_before_shutdown_minutes": [
    "Unknown field."
  ]
} 

 
Visit this link to refer to the ComputeInstance schema if needed: https://aka.ms/ml-cli-v2-compute-aml-yaml-reference.
The easiest way to author a specification file is using IntelliSense and auto-completion Azure ML VS code extension provides: https://code.visualstudio.com/docs/datascience/azure-machine-learning. To set up: https://docs.microsoft.com/azure/machine-learning/how-to-setup-vs-code

```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- create a .yml file with the `idle_time_before_shutdown_minutes: 30` setting
- `az ml compute create -f that.yml`

## Expected Behavior

The compute is created

## Environment Summary
```
macOS-12.6.1-x86_64-i386-64bit, Darwin 21.6.0
Python 3.10.8
Installer: HOMEBREW

azure-cli 2.42.0

Extensions:
ml 2.11.0
account 0.2.5

Dependencies:
msal 1.20.0
azure-mgmt-resource 21.1.0b1
```

**Comment by yonzhan** [comment]
route to CXP team

**Comment by navba-MSFT** [comment]
@vladiliescu Apologies for the late reply. Thanks for reaching out to us and reporting this issue. This bug was fixed in SDK version 1.1.1. The fix will be released as part of  version **2.12.0** early next week. As a temporary workaround, you can use the property in your YAML as such:

**idle_time_before_shutdown: PT{number of minutes}M**

instead of the idle_time_before_shutdown_minutes property. Note that this property will be deprecated in the next CLI release.
